src/pybroker/data.py

```diff
@@ -532,3 +533,98 @@ def _fetch_data(
+        ADJ_CLOSE: Column name of adjusted close prices.
+    """
+
+    ADJ_CLOSE: Final = "adj_close"
+    __TIMEFRAME: Final = "1d"
+
+    def __init__(self):
+        super().__init__()
+        self._scope.register_custom_cols(self.ADJ_CLOSE)
+
+    def query(
+        self,
+        symbols: Union[str, Iterable[str]],
+        start_date: Union[str, datetime],
+        end_date: Union[str, datetime],
+        _: Optional[str] = "",
+    ) -> pd.DataFrame:
+        r"""Queries data from `Yahoo Finance <https://finance.yahoo.com/>`_\ .
+        The timeframe of the data is limited to per day only.
+
+        Args:
+            symbols: Ticker symbols of the data to query.
+            start_date: Start date of the data to query (inclusive).
+            end_date: End date of the data to query (inclusive).
+
+        Returns:
+            :class:`pandas.DataFrame` containing the queried data.
+        """
+        return super().query(symbols, start_date, end_date, self.__TIMEFRAME)
+
+    def _fetch_data(
+        self,
+        symbols: frozenset[str],
+        start_date: datetime,
+        end_date: datetime,
+        _: Optional[str],
+    ) -> pd.DataFrame:
+        """:meta private:"""
+        start_date = to_datetime(start_date).strftime("%Y%m%d")
+        end_date = to_datetime(end_date).strftime("%Y%m%d")
+        symbols = list(symbols)
+        symbols_simple = [item.split('.')[0] for item in symbols]
+        result = pd.DataFrame()
+        for i_num in range(len(symbols)):
+            try:
+                temp_df = akshare.stock_zh_a_hist(symbols_simple[i_num], start_date=start_date, end_date=end_date, period="daily", adjust="")
+                temp_hfq_df = akshare.stock_zh_a_hist(symbols_simple[i_num], start_date=start_date, end_date=end_date, period="daily", adjust="qfq")
+                temp_df['symbol'] = symbols[i_num]
+                temp_df[self.ADJ_CLOSE] = temp_hfq_df['收盘']
+            except KeyError as e:
+                temp_df = pd.DataFrame()
+            result = pd.concat([result, temp_df], ignore_index=True)
+        if result.columns.empty:
+            return pd.DataFrame(
+                columns=[
+                    DataCol.SYMBOL.value,
+                    DataCol.DATE.value,
+                    DataCol.OPEN.value,
+                    DataCol.HIGH.value,
+                    DataCol.LOW.value,
+                    DataCol.CLOSE.value,
+                    DataCol.VOLUME.value,
+                    self.ADJ_CLOSE,
+                ]
+            )
+        if result.empty:
+            return result
+        result.rename(columns={
+            '日期': DataCol.DATE.value,
+            '开盘': DataCol.OPEN.value,
+            '收盘': DataCol.CLOSE.value,
+            '最高': DataCol.HIGH.value,
+            '最低': DataCol.LOW.value,
+            '成交量': DataCol.VOLUME.value,
+            "symbol": DataCol.SYMBOL.value,
+
+        }, inplace=True)
+        result['date'] = pd.to_datetime(result['date'])
```

Comment:
The formatting will fail the tests that run post-commit. Can you run these commands?

1. black (auto format)
2. flake8 (lint)
3. mypy --show-error-codes --ignore-missing-imports (typecheck)

You can install these with:
pip install black flake8 flake8-bugbear mypy

Reply:
ok